### DISCUSSION TITLE
Switching between applications

### DISCUSSION THREAD

**Discussion by raine** [ianyh/Amethyst#1512] [General]
Just wondering, how do people switch between applications when using Amethyst?

Some solutions I'm aware of:

- Plain cmd+tab
- Hammerspoon
- https://contexts.co/

Any other great workflows? Thanks.

**Comment by ianyh** [comment]
I usually use plain old command tab and command command tilde in fullscreen and focus rotation in tall.

**Comment by izeau** [comment]
I’m using both Amethyst’s shortcuts and [AltTab](https://alt-tab-macos.netlify.app/). AltTab is great because it allows switching between windows — not apps.

**Comment by mmoulou** [comment]
I added a shortcut for each workspace, as an application always uses the same workspace. I also added an Amethyst shortcut to switch between windows in the same workspace.

If you use AltTab, you need to press Alt+Tab several times to switch to your application
